ez-rp.py

```diff
@@ -10,16 +10,20 @@
             [sg.Text("                                                                                                                                                                                                                                                                                 ", key="ERROR", text_color="red")], #This string is long.
             [sg.Text('Enter your client ID', key="~CLIENTID~"), 
             sg.InputText()],
+            [sg.Text('Large Image ID:', key="~LIMAGEID~"),
+            sg.InputText()],
+            [sg.Text('Small Image ID:', key="~SIMAGEID~"),
+            sg.InputText()],
             [sg.Text('Enter state of Rich Presence (Top text)', key="~RPCSTATE~"), 
             sg.InputText()],
             [sg.Text('Enter details of Rich Presence (2nd to bottom text)', key="~RPCDETAILS~"), 
             sg.InputText()],
-            [sg.Text('(Optional) First button text', key="~firstbuttontext~"), 
+            [sg.Text('(Optional) First button text', key="~firstbuttontext~", visible = True), 
             sg.InputText()],
-            [sg.Text('(Mandatory if you put text for first button) First button URL', key="~firstbuttonurl~"), 
+            [sg.Text('(Mandatory if you put text for first button) First button URL', key="~firstbuttonurl~", visible = True), 
             sg.InputText()],
-            [sg.Button('Run!'), sg.Button('Cancel'), sg.Button('Join Discord!')],
-            [sg.Text('Ravens Rich Presence Generator is a free open source program to generate custom discord rich presences. If anyone tries to sell you Ravens Rich Presence Generator, or tries to redistribute it as their own, please report it immediately. You are being scammed')]
+            [sg.Button('Run!'), sg.Button('Cancel'), sg.Button('Join Discord!'), sg.Button('Help'),],
+            [sg.Text('Ravens Rich Presence Generator is a free open source program to generate custom discord rich presences. \n If anyone tries to sell you Ravens Rich Presence Generator, or tries to redistribute it as their own, please report it immediately.\n You are being scammed', justification='center')]
```

Comment:
added help button and made the disclaimer thingy more readable on smaller screens